tools/prune-old-builds.py

```diff
@@ -0,0 +1,81 @@
+import os
```

Comment:
Nit: Put a short copyright notice at the top of the file too. (See other files for examples.)